src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -58,13 +59,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
       return;
     }
 
-    // TODO(JW): Merge "the creation process" from here to UserStore
     String password = request.getParameter("password");
-    String hashedPassword = BCrypt.hashpw(password, BCrypt.gensalt());
-
-    User user = new User(UUID.randomUUID(), username, hashedPassword, Instant.now());
-    userStore.addUser(user);
 
+    userStore.addUser(username, password, false);
```

Comment:
You can add the `/*admin=*/` comment here too.

Reply:
Fixed!